icu4c/source/test/intltest/dtifmtts.cpp

```diff
@@ -905,14 +905,14 @@ void DateIntervalFormatTest::testFormat() {
         "de", "CE 2007 10 10 10:10:10", "CE 2008 10 10 10:10:10", "dMy", "10.10.2007\\u2009\\u2013\\u200910.10.2008",
 
 
-        "de", "CE 2007 10 10 10:10:10", "CE 2008 10 10 10:10:10", "My", "10.2007\\u2009\\u2013\\u200910.2008",
+        "de", "CE 2007 10 10 10:10:10", "CE 2008 10 10 10:10:10", "My", "10/\\u20092007\\u2013\\u200910/2008", // bad interval pattern
```

Comment:
is this tracked on cldr if it's bad

Reply:
> de changed formats for yM to use '/' instead of '.' (and made an error one of them, to be fixed with https://unicode-org.atlassian.net/browse/CLDR-15864

Reply:
Yes it is already fixed on the CLDR side (but after the tag for this integration) per xxxx (will fill in shortly)